src/main/webapp/app/course/manage/detail/course-detail.component.html

```diff
@@ -40,6 +40,13 @@
                             [currentAbsolute]="courseDTO?.currentAbsoluteAverageScore"
                             [currentMax]="courseDTO?.currentMaxAverageScore"
                         />
+                        <jhi-course-detail-doughnut-chart
```

Comment:
Can you add a condition here to only show the chart if Iris/Athena/LLM-stuff is active on the server/course?